### PR TITLE
Added an example to `ipfs id`

### PR DESCRIPTION
**Pull request by RichardLitt**
I was super confused how to use the `-f` flag. This helped, but I had to go into
the test files to find it. Added it here.

License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

**Comment by whyrusleeping** [comment]
any day now, the travis tests will finish...

File: core/commands/id.go [hunk 1]
```diff
@@ -49,6 +49,10 @@ If no peer is specified, prints out information for local peers.
 <pver>: Protocol version.
 <pubkey>: Public key.
 <addrs>: Addresses (newline delimited).
+
+Example:
+
+		$ ipfs id Qmece2RkXhsKe5CRooNisBTh4SK119KrXXGmoK6V3kb8aH -f="<addrs>\n"
```
Comment (dignifiedquire) [inline core/commands/id.go]: Isn't that a bit much indentation?

**Comment by RichardLitt** [comment]
Fixed. Also removed $.

**Comment by dignifiedquire** [comment]
LGTM

**Comment by whyrusleeping** [comment]
Cool, LGTM

**Pull request merged**